core/os/file/file.go

```diff
@@ -42,7 +43,11 @@ func RemoveAll(f Path) error {
 
 // Symlink creates a new symbolic-link at link to target.
 func Symlink(link, target Path) error {
-	return os.Symlink(target.value, link.value)
+	if rel_path, err := filepath.Rel(link.value+"/..", target.value); err == nil {
```

Comment:
I will double-check but this should support both. (I will also test with Parent())